config/locales/es.yml

```diff
@@ -0,0 +1,25 @@
+es:
+  devise:
+    omniauth_callbacks:
+      success: "Sesión realizada con tu cuenta de %{kind}."
+  spree:
+    user_was_not_valid: "El usuario no es válido."
+    add_another_service: 'Añadir otro servicio para inicio de sesión:'
+    authentications:
+      destroy: 'Método de autenticación eliminado satisfactoriamente.'
+    back_to_authentication_methods_list: "Volver a la lista de métodos de autenticación"
+    edit_social_method: 'Editando metodo de autenticación'
+    new_social_method: 'Nuevo método de autenticación'
+    no_authentication_methods_found: "No se encuentran ningún método de autenticación"
```

Comment:
I'd rephrase this, something like: `No hay métodos de autenticación disponibles.`, what do you think?

Reply:
ok